lib/Doctrine/DBAL/Driver/IBMDB2/DB2Connection.php

```diff
@@ -175,4 +175,18 @@ public function errorInfo()
             1 => $this->errorCode(),
         );
     }
+
+    /**
+     *
+     * Retrieves ibm_db2 native resource handle.
+     *
+     * Could be used if part of your application is not using DBAL.
+     *
+     * @return resource
+     */
+    public function getConn() //getWrappedResourceHandle
```

Comment:
Should probably keep the same naming then, but still, we'd need a specific use-case example for this. I don't see why a user can't simply build a new connection object rather than relying on doctrine's.